### ISSUE TITLE
Block events don't register the correct world

### ISSUE THREAD

**Issue by brandonganem** [splunk/minecraft-app#9]
[13:13:02 INFO]: name=PlayerEvent event_id= game_time=5282 world=yyy x=-123 y=130 z=283 action=move player=xxxx from_x=-123 from_y=130 from_z=284 to_x=-123 to_y=130 to_z=283
[13:13:03 INFO]: name=BlockEvent event_id= game_time=3437293 world=DEFAULT x=-122 y=132 z=284 action=block_broken block_type=WOOD_STAIRS player=xxxxx base_type=WOOD_STAIRS

These blocks are being placed in a named world, not "DEFAULT"

**Comment by brandonganem** [comment]
Could be related to #10 in that this piece of code isn't handling a world name with a space well?

**Comment by TheProducer835** [comment]
I don't think so. PlayerEvent world=<world name> Block Event world=DEFAULT implies the former is asking for name and the latter is asking for worldType as if it's using the wrong module for the value. can you take a look?

**Comment by PowerSchill** [comment]
I have implemented a fix via PowerSchill/minecraft-app@235ed45063c5e1c56bfd2b741cc4db4d85f3dc75 

The worlds are currently world, world_the_nether, and world_the_end for overworld, the nether, and the end respectively.
